cmd/notation/cert/add.go

```diff
@@ -28,6 +28,14 @@ func certAddCommand(opts *certAddOpts) *cobra.Command {
 			opts.path = args
 			return nil
 		},
+		Long: `Manage certificates in trust store
+
+Example - Add certificates with type "ca" to the trust store "acme-rockets":
+  notation cert add --type ca --store "acme-rockets" $XDG_CONFIG_HOME/notation/truststore/x509/ca/acme-rockets/
```

Comment:
```suggestion
  notation certificate add --type ca --store acme-rockets ./acme-rockets.crt
```
(There's no need to add double quotation around the store name)

Reply:
Updated